platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -112,6 +112,8 @@ typedef	struct {
 					_ipc_map_remote_pool() */
 } _ipc_pktio_t;
 
+#define PKTIO_PRIVATE_SIZE 384
```

Comment:
Can this be a computed value. _e.g._, `sizeof` some union? Fixed sizes like this seem error-prone in the face of potential future growth.

Reply:
Well. The whole idea is to have generic space with all unions hidden in the file implementations. Another approach might be to have `void *` field here and let individual drivers allocate private data from internal pools.

Reply:
Allocation would require indirection, which is undesirable on fast paths. Since you're guarding this with `ODP_STATIC_ASSERT()` in each of the modules it won't cause overruns, so you're OK there. I'm just thinking more about futures when some may not be as careful.